helm/robusta/values.yaml

```diff
@@ -94,6 +94,18 @@ builtinPlaybooks:
   - image_pull_backoff_reporter:
       rate_limit: 3600
 
+- triggers:
+  - on_job_failure: {}
+  actions:
+  - create_finding:
```

Comment:
None of these events create a finding, they only add enrichments so without this we get a "generic changed" finding without any additional information like service_key, subject name, kind, namespace ...